Sidebar - Documentation Project settings Applications (#63)
ynput/ayon-documentation: website/docs/project_settings/project_settings_applications.md

---
id: project_settings_applications
title: Applications
sidebar_label: Applications
---
# Show only available applications

> Enable it to show on Ftrack and the Launcher only the applications available on the computer.

Reviewer: ```suggestion
When enabled, the Launcher and Ftrack actions only show the applications available on the computer.
```
